.github/workflows/dev_nbval.yml

```diff
@@ -0,0 +1,214 @@
+name: nbval (develop)
```

Comment:
I didn't get why this PR contains changes that do not relate to the topic. I recommend reshuffling of the structure/CI in the separate PRs.

Reply:
it looks like rebase issue

Reply:
fixed